samples/unit-tests/series-arearange/arearange/demo.js

```diff
@@ -1,3 +1,24 @@
+QUnit.test('Fill opacity zero(#4888)', function (assert) {
```

Comment:
Not related to `arearange` series. Create a new demo set, `unit-tests/series-area/fillopacity`. Verify it in utils.